spec.bs

```diff
@@ -649,19 +649,24 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
     1. Run [=update bid counts=] with |bidIgs|.
     1. Run [=interest group update=] with |auctionConfig|'s
       [=auction config/interest group buyers=].
+1. Let |generateBidResults| be a new [=list=] of [=generate bid results=].
+1. Let |auctionReporter| be a new [=auction reporter=].

```

Comment:
Since this is just an object that holds data, I suggest calling it auctionReports (and same for the class name).  Or could add "Info" or "Data" or something to the end instead of the "s".